examples/notebooks/parameter-management.ipynb

```diff
@@ -0,0 +1,397 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Parameter management\n",
+    "\n",
+    "PyBamm comes with a set of pre-defined parameters for common chemistries. These files are located inside your PyBaMM installation directory, typically something like\n",
+    "```\n",
+    "../venv/lib/pythonX.Y/site-packages/pybamm/input/parameters\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The `input/parameters` directory is organised as follows:\n",
+    "```\n",
+    "input/parameters\n",
+    "  lithium-ion/ # chemistry\n",
+    "    anodes/ # Component\n",
+    "      graphite_Chen2020/ # parameter set\n",
+    "        parameters.csv\n",
+    "\t    graphite_LGM50_diffusivity_Chen2020.py\n",
+    "        ...\n",
+    "      graphite_mcmb2528_Marquis2019/\n",
+    "      graphite_Ecker2015/\n",
+    "      ...\n",
+    "    cathodes/\n",
+    "    cells/\n",
+    "    electrolytes/\n",
+    "    seis/\n",
+    "    separators/\n",
+    "    experiments/\n",
+    "  lead-acid/\n",
+    "    ...\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "When setting parameter values using `pybamm.ParameterValues` and a filename, the file is searched in several locations, described by\n",
+    "the variable `pybamm.PARAMETER_PATH`.\n",
+    "The default is "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "['/home/tlestang/projects/PyBaMM/examples/notebooks',\n",
+       " '/home/tlestang/virtualenvs/pybamm/lib/python3.7/site-packages/pybamm/input/parameters']"
+      ]
+     },
+     "execution_count": 1,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# In directory /tmp\n",
```

Comment:
maybe we should remove this comment as it returns `blah/blah/PyBaMM/examples/notebooks`, not `/tmp`

Reply:
Oh good point